assets/js/blocks/price-filter/block.js

```diff
@@ -36,9 +37,24 @@ const PriceFilterBlock = ( { attributes, isEditor = false } ) => {
 	const [ minPrice, setMinPrice ] = useState();
 	const [ maxPrice, setMaxPrice ] = useState();
 
+	const priceRangeData =
+		typeof results.price_range !== 'object'
+			? {
+					currency_code: 'USD',
+					currency_symbol: '$',
+					currency_minor_unit: 2,
+					currency_decimal_separator: '.',
+					currency_thousand_separator: ',',
+					currency_prefix: '$',
+					currency_suffix: '',
+					min_price: undefined,
+					max_price: undefined,
+			  }
```

Comment:
This fallback should be defined outside the component so that the object is only created once.